src/sqlfluff/cli/commands.py

```diff
@@ -769,6 +769,35 @@ def do_fixes(
     return False  # pragma: no cover
 
 
+def _handle_unparsable(
+    fix_even_unparsable: bool,
+    initial_exit_code: int,
+    linting_result: LintingResult,
+    formatter,
```

Comment:
can we add a type here?

Reply:
Hmmmmm good question. We haven't typically typed that before - but it's a good shout that we probably should. I'll see if I can make that work 🤔

Reply:
@WittierDinosaur - yes I can! Sorted now